viral/offline_sequence_detection.py

```
@@ -0,0 +1,1255 @@
+    ppm[3, 5] = 0.7
+    return ppm
+
+
+if __name__ == "__main__":
+    for mouse_name in SESSIONS_KEEP.keys():
+        for stage, date in SESSIONS_KEEP[mouse_name].items():
+            print(f"Processing {mouse_name} - {stage} - {date}")
+            if date:
+                try:
+                    main(mouse_name, date)
+                except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)